Orange/canvas/application/canvasmain.py

```diff
@@ -539,13 +505,14 @@ def setup_actions(self):
                     )
 
         self.show_output_action = \
-            QAction(self.tr("Show Output View"), self,
-                    toolTip=self.tr("Show application output."),
-                    triggered=self.show_output_view,
+            QAction(self.tr("Show Output Dock"), self,
+                    toolTip=self.tr("Show application standard output."),
+                    checkable=True,
+                    triggered=lambda checked: self.output_dock.setVisible(checked),
                     )
 
         self.show_report_action = \
-            QAction(self.tr("Show Report View"), self,
+            QAction(self.tr("Show Report Window"), self,
```

Comment:
I would rename this option to Report. Show is redundant as this action is located in the View menu and can be considered as View > Report.